### ISSUE TITLE
[BUG] Manifest V3 loses tablist after inactivity

### ISSUE THREAD

**Issue by OhMyGuus** [OhMyGuus/I-Still-Dont-Care-About-Cookies#362]
Manifest V3 (chromium browsers) loses the tablist after a bit of inactivity, this might be a bug that causes warnings/popups showing up on websites.

**Comment by nicolaasjan** [comment]
Do you mean that the options don't show up on left click?

This weird behaviour was also reported in the [thread about IDCAC](https://www.wilderssecurity.com/threads/avast-aquired-i-dont-care-about-cookies.447558/) on [Wilderssecurity.com](https://www.wilderssecurity.com), starting [here](https://www.wilderssecurity.com/threads/avast-aquired-i-dont-care-about-cookies.447558/page-5#post-3113828).

Confirmed (Chromium and Vivaldi) on https://www.cnbc.com/ and https://www.beursgorilla.nl/
A reload makes the tabs appear again.

Also, on [beursgorilla.nl](https://www.beursgorilla.nl/) there is no green check mark (there is one on Firefox and also on Chromium with the MV2 version built from source).

**Comment by OhMyGuus** [comment]
> Do you mean that the options don't show up on left click?
> 
> This weird behaviour was also reported in the [thread about IDCAC](https://www.wilderssecurity.com/threads/avast-aquired-i-dont-care-about-cookies.447558/) on [Wilderssecurity.com](https://www.wilderssecurity.com), starting [here](https://www.wilderssecurity.com/threads/avast-aquired-i-dont-care-about-cookies.447558/page-5#post-3113828).
> 
> Confirmed (Chromium and Vivaldi) on https://www.cnbc.com/ and https://www.beursgorilla.nl/ A reload makes the tabs appear again.
> 
> Also, on [beursgorilla.nl](https://www.beursgorilla.nl/) there is no green check mark (there is one on Firefox and also on Chromium with the MV2 version built from source).
> 
> Bug still present in build from latest source (commit [f2d76fd](https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/commit/f2d76fdf5af8f4d17727fb9c148c844a022ad4b1)).

Yup, the left click on the extension without showing the hostname is part of the same bug.  The bug is because of service workers lifetime which shouldn't be to hard to fix

About the green checkmark on websites that use request blocking is quite normal since that part needs to be rewritten for manifest v3 (There isn't a event for it yet so need to think about another good way to detect if it blocked a request)

**Comment by nicolaasjan** [comment]
This is why I use MV2 extensions whenever possible for as long as they work. 😀️

Hope you will keep the MV2 option alive for browsers that (hopefully) keep supporting it.
